.history/src/js/nizovi_20200712154956.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-unused-vars */
+// 1. zadatak
+const slova = ['a', 'b', 'c']
+// 2. zadatak
+const brojevi = [1, 2, 3]
+// 3. zadatak
```

Comment:
Mogao bi da kopiraš zadatke u nov fajl na ovoj grani da bi mogao da dobiješ review za njih. Za ono što je na masteru ne možemo da radimo review.

Reply:
Mozda glupo, ali jel moze pojasnjenje..

Reply:
U sustini, na ovoj grani, na ovom pull request, mozemo da vidimo samo ono sto si menjao od momenta kad si je napravio. Kad udjes u files changed, videces da se vidi samo zadatak 4 fazon, jer si na njemu menjao nesto. Sve ostalo je na masteru, a na master grani se ne radi pull request, samim tim ni review. Mozda sam te zbunio jos vise sad :D 
Samo na VScode napravi novi fajl i budi na ovoj grani, onda prekopiraj sve zadatke koje si uradio u taj novi fajl i onda njega git add ./git commit/git push